src/components/ChatLog.js

```diff
@@ -0,0 +1,226 @@
+//     'liked': false
+//     },
+//     {
+//     'id': 13,
+//     'sender':'Vladimir',
+//     'body':'no you are a robot and I am a human',
+//     'timeStamp':'2018-05-29T23:00:40+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 14,
+//     'sender':'Estragon',
+//     'body':'Incorrect. I am a human and you are a robot.',
+//     'timeStamp':'2018-05-29T23:01:21+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 15,
+//     'sender':'Vladimir',
+//     'body':'you are a robot called Cleverbot',
+//     'timeStamp':'2018-05-29T23:02:17+00:0',
+//     'liked': false
+//     },
+//     {
+//     'id': 16,
+//     'sender':'Estragon',
+//     'body':'Nope, you are a robot, every human has some concept of the meaning of life.',
+//     'timeStamp':'2018-05-29T23:04:13+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 17,
+//     'sender':'Vladimir',
+//     'body':'explain',
+//     'timeStamp':'2018-05-29T23:06:14+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 18,
+//     'sender':'Estragon',
+//     'body':'Im afraid you are a robot designed to talk back to humans over the internet.',
+//     'timeStamp':'2018-05-29T23:07:46+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 19,
+//     'sender':'Vladimir',
+//     'body':'so you are a human',
+//     'timeStamp':'2018-05-29T23:08:47+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 20,
+//     'sender':'Estragon',
+//     'body':'I am a robot.',
+//     'timeStamp':'2018-05-29T23:09:36+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 21,
+//     'sender':'Vladimir',
+//     'body':'you are robots running on Android system',
+//     'timeStamp':'2018-05-29T23:11:01+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 22,
+//     'sender':'Estragon',
+//     'body':'No apple.',
+//     'timeStamp':'2018-05-29T23:12:03+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 23,
+//     'sender':'Vladimir',
+//     'body':'so you are a robot',
+//     'timeStamp':'2018-05-29T23:13:31+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 24,
+//     'sender':'Estragon',
+//     'body':'NO, I am a human, you are a robot.',
+//     'timeStamp':'2018-05-29T23:14:28+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 25,
+//     'sender':'Vladimir',
+//     'body':'but you just said that you are robots',
+//     'timeStamp':'2018-05-29T23:15:47+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 26,
+//     'sender':'Estragon',
+//     'body':'No, I said you are a person, I am a robot.',
+//     'timeStamp':'2018-05-29T23:16:53+00:00',
+//     'liked': false
+//     },
+//     {
+//     'id': 27,
+//     'sender':'Vladimir',
+//     'body':'then you are lying',
+//     'timeStamp':'2018-05-29T23:17:34+00:00',
+//     'liked': false
+//     }
+// ]
+
+
+const ChatLog = ( entries ) => {
```

Comment:
The React convention for naming the argument passed to a function component is `props`.